shared/src/business/utilities/getDocumentTitle.js

```diff
@@ -0,0 +1,30 @@
+const {
+  formatDocketEntry,
+  getFilingsAndProceedings,
+} = require('./getFormattedCaseDetail');
+
+/**
+ * Gets document title based on documentTitle and additionalInfo fields
+ *
+ * @param {object} applicationContext the applicationContext
+ * @param {object} docketEntry the docketEntry
+ * @returns {object} the document title
+ */
+
+const getDocumentTitle = ({ applicationContext, docketEntry }) => {
+  let { documentTitle } = docketEntry;
+  if (docketEntry.addToCoversheet) {
+    if (docketEntry.additionalInfo) {
+      documentTitle += ` ${docketEntry.additionalInfo}`;
```

Comment:
I think this could be changed to
```suggestion
      documentTitle += ` ${docketEntry.additionalInfo || ''}`.trim(); 
```
to remove the if statement wrapper, although there could be a better way I'm not familiar with.

Reply:
Updated!